### ISSUE TITLE
Acer Nitro 5 does not work as expected with NFBC

### ISSUE THREAD

**Issue by renanwp2** [hirschmann/nbfc#815]
**Describe the wonderfulness**
The "Acer Predator G3-572" configuration works pretty well on "Acer Nitro 5 AN515-52-75Q8". The GPU and CPU fans behave pretty well on all applications I could see for now (casual usage, gaming and benchmark). The output of command "nbfc status" are particulary close to the reality and the fans looks to behave pretty well. I suggest you modify the critical temp to 75ºC. The fans are never turned off, as expected. As far I could see, the all application consumed about 50Mb of RAM! Well... If it is not the solution, at least will lead to the correct solution. I hope this helps other people to know understand the EC from the Acer Nitro 5 series, cos I dont. Hahaha

**To Reproduce**
Follow the guide presented on "https://github.com/hirschmann/nbfc/wiki/First-steps" and pay attention to the NEW stable version of Mono and not from your repositories that may be outdated as mine was.

**Expected behavior**
Happiness

**System information**
 - Debian 10.2 XFCE (Buster stable)
 - NBFC 1.6.3.0.

**Additional context**
My special thanks to all contributors.

**Comment by renanwp2** [comment]
The other predator worked for me too, but I modified the file to drop the temperatures down in a faster way. If the site dows not lie, here is my new configuration : [https://textuploader.com/16a66](url)

**Comment by renanwp2** [comment]
My fears: this cause issues due the fact that "Acer Predator G3-572" works fan in a higher velocity than mine and demage my system. There is this possibility?

**Comment by renanwp2** [comment]
Yeah! Put the critical temp at low is an problem! The kernel is pointing out that I am suffering throttling.

**Comment by renanwp2** [comment]
No it does not anymore. At first glanse looked light, but after some time, the kernel are accusing CPU throtling even though there is no visible heat.

**Comment by renanwp2** [comment]
I'm passing throu some problems right now. I'm not totally capable of distinguish if it was a problemofmy hardware or the NFBC. Wail for updates. Developer says: 

Is there a way to reset my notebook if something went wrong?
Usually, if you disable the fan control service, it will reset the EC (which controls the fans).
If this does not work, you can reset the EC manually.

If the notebook has a removable battery:
Turn off the notebook, unplug it and remove the battery for about 20-30 seconds.

In case the battery is not removable:
Turn off the notebook, unplug it and press and hold the power button for 10-15 seconds. If that doesn't work, search in your notebook's user manual for a way to reset it.

**Comment by renanwp2** [comment]
Update:
Fortunately, everything is right now. My suggestion to the developer is put an aware on first steps ou create one page calling "usage warnings". Mainly reporting some troubles that one could pass and reccomended actions in that case. 
What I passed through? I passed through overheating even though the fans was at max speed! My kernel was throuing awarnings on my screen of thermal throttle! That's too aggressive! Take care on probing your EC end at first oportunity, please reset your EC as suggested by the author. =) 
On Acer laptops, there is an thin hole that you put some needle in it and the batters will be "removed" and reset the EC to factory. So... you are an Acer user that is the solution. So... right now... I wont use that software increase my fan speed and I dont recommend no one to do so. Anyway, the developer has done a great job here!

**Comment by renanwp2** [comment]
My suggestion to author is to alert people that this software need some libs from playonlinux too!

**Comment by renanwp2** [comment]
After testes, notebook is normal! So.. that's it! Try to put the max temp to 100ºC and make  a good fan curve and voila! You are welcome. =) Hope this helpes people in future! Performance is yet good and my system is working properly without any NBFC. I don't live my vendor's fans curve, but I will try it when someone will be capable of make a good configuration for me. =)

**Comment by renanwp2** [comment]
Ok! After installing Windows  I discovered that the configs are exacly the same! I even didn't get how the fuck the creator of Acer Predator G3-572 (john hancock'ed as Chandradeep Dey) knew that ReadWriteWords is true! I have no clue about this parameter, but about others one, I'm pretty sure that they are correct. Something fun after the instalation of this program is that the reported valocity of my fan is set to 6122, but the original one is 5172. Other thing is that the reported temperature of id 0 temperature is wrong on Linux I guess. On Linux the temperature goes to 90ºC easily, but not on Windows never passed 80ºC. For now, I dont know I if the "side effect" that I suffered was caused by the NBFC or by my imprudent ec-proc tests. But... now I will reinstall my drivers on Windows and see If my configs goes back to original and don't suffer more cpu throttling on Linux.
Just in case, Acer Nitro 5 users, try EC reset if something goes wrong.
